### PR TITLE
Add silent message update support

### PR DESCRIPTION
**Pull request by yinan-symphony**
- Update Symphony APIs spec version

 - Add silent flag support according to the new spec.
   When silent is true, the new updated message is marked as read, otherwise is unread.
   True is the default value.

### Description
Closes #[ISSUE NUMBER]

Please put here the intent of your pull request.

### Dependencies
List the other pull requests that should be merged before/along this one.

### Checklist
- [ ] Referenced an issue in the PR title or description
- [ ] Filled properly the description and dependencies, if any
- [ ] Unit tests updated or added
- [ ] Docstrings added or updated
- [ ] Updated the documentation in [docsrc folder](../tree/main/docsrc)

### CODE REVIEW

File: api_client_generation/generate.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ code_gen_dir=`pwd`
 project_root=$code_gen_dir/..
 echo $code_gen_dir
 
-commit_hash=136b530512eea112de73591906051871d034750a
```
Comment (symphony-soufiane) [inline api_client_generation/generate.sh:7]: When we update this commit hash, we might want to run the sh script to update the generated code. See here https://github.com/finos/symphony-bdk-python/blob/main/docsrc/markdown/tech/generated_api.md

Comment (symphony-soufiane) [reply api_client_generation/generate.sh:7]: And include the new generated files in the PR of course.

File: symphony/bdk/core/service/message/message_service.py [hunk 2]
```diff
@@ -466,7 +466,7 @@ async def search_messages_one_page(skip, limit):
 
     @retry
     async def update_message(self, stream_id: str, message_id: str, message: Union[str, Message], data=None,
-                             version: str = "") -> V4Message:
+                             silent=True, version: str = "") -> V4Message:
```
Comment (symphony-soufiane) [inline symphony/bdk/core/service/message/message_service.py]: I'm wondering if adding this parameter doesn't break the backward compatibility. I would rather add a new method that takes this new argument, and I call it in this one with the default silent value (True).

Comment (yinan-symphony) [reply symphony/bdk/core/service/message/message_service.py]: i checked this point actually. this `silent=True` syntax in python is working as an optional arg with the default value. so user can choose to specify it or simply ignore it, in this case the default value is applied.
you can see that the unit test does not change the signature, but it is still working.

File: api_client_generation/generate.sh (hunk 1 above)
Comment (yinan-symphony) [reply api_client_generation/generate.sh:7]: sure, i am doing it now

Comment (yinan-symphony) [reply api_client_generation/generate.sh:7]: @symphony-soufiane done.

**Review by symphony-soufiane — APPROVED** [review]

**Pull request merged**